tasks/e2e-simple.sh

```diff
@@ -314,6 +317,9 @@ npm link "$root_path"/packages/eslint-config-react-app
 npm link "$root_path"/packages/react-dev-utils
 npm link "$root_path"/packages/react-scripts
 
+# Test lint
+"$root_path"/packages/react-scripts/bin/react-scripts.js lint
```

Comment:
This doesn't make sense to me. What is this testing?

After you eject, `react-scripts` doesn't exist (it might on CI but you shouldn't rely on this—the point of ejecting is to wipe out `react-scripts` on the next installs).

So the script clearly doesn't "still work" after ejecting. If you had `"lint": "react-scripts lint"` before ejecting, it will stop working after.